docs/content/docs/try-flink-ml/java/quick-start.md

```diff
@@ -54,23 +63,17 @@ export FLINK_HOME=`pwd`
     available)
 ### Build Flink ML library
 
-In order to use Flink ML's CLI you need to have the latest binary distribution
-of Flink ML. You can acquire the distribution by building Flink ML's source code
-locally with the following command.
-
-```bash
-cd ${path_to_flink_ml}
-mvn clean package -DskipTests
-cd ./flink-ml-dist/target/flink-ml-*-bin/flink-ml*/
-```
+Please walk through this [guideline]({{< ref "docs/development/building" >}}) to
+build Flink ML's Java SDK.
 
 ### Add Flink ML binaries to Flink
 
 You need to copy Flink ML's binary distribution files to Flink's folder for
-proper initialization. Please run the following command from Flink ML's binary
-distribution's folder.
+proper initialization. Please run the following command from Flink ML's root
+folder.
 
 ```bash
+cd ./flink-ml-dist/target/flink-ml-*-bin/flink-ml*/
```

Comment:
For master branch where the jars are compiled from source, it might be better to do this:

```
cd ${path_to_flink_ml}
cp ./flink-ml-dist/target/flink-ml-*-bin/flink-ml*/lib/*.jar $FLINK_HOME/lib/
```

For released branch, we probably want to let users download the jars into this directory.